python/python/Ice/ObjectPrx.py

```diff
@@ -0,0 +1,727 @@
+
+import IcePy
+
+class ObjectPrx(IcePy.ObjectPrx):
```

Comment:
I would prefer to use delegation to the C++ object, like for example Communicator. Am I missing something?